pkg/resources/stream.go

```diff
@@ -170,6 +177,7 @@ func CreateStream(d *schema.ResourceData, meta interface{}) error {
 
 	builder.WithOnTable(resultOnTable.DatabaseName, resultOnTable.SchemaName, resultOnTable.OnTableName)
 	builder.WithAppendOnly(appendOnly)
+	builder.WithShowInitialRows(showInitialRows)
```

Comment:
you need to add this prop to the Read func.

For the update side of things, is this property "updateable"? If not, you might want to "ForceNew" on the Resource schema

Reply:
Seems like the only thing you can update are comments so this will require a ForceNew (in fact, every property except comment should be this way)

Reply:
https://docs.snowflake.com/en/sql-reference/sql/alter-stream.html